joomla-projects/joomla-es6
# Remove inline Javascript for administrator/components/com_joomlaupdate/tmpl/joomlaupdate/default.php

Pull Request for Issue https://github.com/joomla-projects/joomla-es6/issues/55.

### Summary of Changes
 Remove inline Javascript for administrator/components/com_joomlaupdate/tmpl/update/joomladefault.php


### Test instructions

If you like to test go to com_joomlaupdater and see in debugger, that the variables <tt>joomlaTargetVersion</tt> is set.
